modules/apps/roll-dialog/weapon-dialog.js

```diff
@@ -120,14 +120,18 @@ export default class WeaponDialog extends AttackDialog {
 
     if (this.item.offhand.value && !this.item.twohanded.value && !(this.item.weaponGroup.value == "parry" && this.item.properties.qualities.defensive)) 
     {
-      this.fields.modifier += -20
-      this.tooltips.addModifier(-20, game.i18n.localize("SHEET.Offhand"))
+      this.computeAmbidextrous()
```

Comment:
I would like to add this additiona method, because this way i may override it in one of my house rules.